### DISCUSSION TITLE
rp6502-sdk renamed?

### DISCUSSION THREAD

**Discussion by mickstone** [picocomputer/community#103] [Q&A] [answered]
Hi All,

Very new to programming, not so new to hardware. I have a working PicoComputer and am now trying to work my way getting my development environment working.
I noticed after creating a new repository based on the vscode-cc65 that the rp6502-sdk is no longer there or renamed as tools.

I'm hoping that someone can just confirm that this is correct as I have no confidence in getting this all set up and working on the software side.

Thanks

**Comment by vruumllc** [comment, marked as answer]
Yes, all you need is the tools directory. 
Follow the directions carefully in the readme for https://github.com/picocomputer/vscode-cc65. and you should be good to go.

**Reply by mickstone** [reply]
Thank you
